ios-base/Extensions/UIViewExtension.swift

```diff
@@ -21,6 +21,26 @@ extension UIView {
     clipsToBounds = true
     layer.cornerRadius = cornerRadius
   }
+  
+  var typeName: String {
+    return String(describing: type(of: self))
+  }
+  
+  func instanceFromNib(withName name: String) -> UIView? {
+    return UINib(nibName: name,
+                 bundle: nil).instantiate(withOwner: self,
+                                          options: nil).first as? UIView
+  }
+  
+  func addNibView(withName nibName: String? = nil) -> UIView? {
```

Comment:
🙌

Reply:
Good one! can we add the autoresizing mask to a param with defaults to `[.flexibleHeight, .flexibleWidth]`?